rocketpy/rocket/rocket.py

```diff
@@ -642,14 +642,14 @@ def add_motor(self, motor, position):
         self.motor_position = position
         _ = self._csys * self.motor._csys
         self.center_of_propellant_position = (
-            self.motor.center_of_propellant_mass - self.motor.nozzle_position
-        ) * _ + self.motor_position
+            self.motor.center_of_propellant_mass * _ + self.motor_position
+        )
         self.motor_center_of_mass_position = (
-            self.motor.center_of_mass - self.motor.nozzle_position
-        ) * _ + self.motor_position
+            self.motor.center_of_mass * _ + self.motor_position
+        )
         self.motor_center_of_dry_mass_position = (
-            self.motor.center_of_dry_mass_position - self.motor.nozzle_position
-        ) * _ + self.motor_position
+            self.motor.center_of_dry_mass_position * _ + self.motor_position
+        )
```

Comment:
I know this PR is meant to just solve the BUG, so I agree this is already complete.
However, I must comment in advance that in the future we might need add a `self.nozzle_position` attribute to the Rocket class, just for convenience.

Reply:
Thank you for the reminder. I still think that we should not add this attribute preemptively in this PR, when the need comes the feature is rather simple and the test may be updated accordingly.